index.html

```diff
@@ -12,6 +12,10 @@
     <script src="./common.js" class="remove"></script>
     <script class="remove"
       src="https://cdn.jsdelivr.net/gh/digitalbazaar/respec-vc@2.0.0/dist/main.js"></script>
+    <script class="removeOnSave"
+      src="https://unpkg.com/reqlist/lib/reqlist.js"></script>
+    <link class="removeOnSave" rel="stylesheet" type="text/css"
+      href="https://unpkg.com/reqlist/lib/reqlist.css" />
```

Comment:
As a warning, reqlist doesn't always capture all the normative statements. It will capture full sentences with normative language in it, but sometimes misses important clarifications in the sentences that follow. It always requires an Editor to make a pass through to make sure we got everything.

We also don't normally include reqlist as a part of non-editorial versions of the spec. Is there a particular reason you want this list to show up in every version of the spec that's published to TR-space?

Reply:
I want to be able to see only normative statements easily, what is the best was to get that done?

I recall we had this turned on for DID Core and then it got turned off... I assume we same will happen here.

Reply:
> I want to be able to see only normative statements easily, what is the best was to get that done?

The way you're doing (on a local branch) it is a good first pass, just don't trust it to get all of the statements. What I tend to do is use reqlist to do the first pass right before CR, move them into a spreadsheet, and then had check the questionable looking normative statements by hand. 

Keeping the list in the spec and publishing it from now until CR tends to not be the best way to get people to look at the list because it can be janky, and people tend to not look at janky things more than a few times (because they end up not trusting the output).

> I recall we had this turned on for DID Core and then it got turned off... I assume the same will happen here.

Yeah, we can do that too, with the caveat/warning above.

To be concrete, add the lines of code you're adding in this PR, but then comment it out using HTML and only uncomment it when  you need it.

OR, you might find that reqlist creates a perfect list of normative requirements for our spec, in which case, we could turn it on and keep it turned on.